wcfsetup/install/files/lib/data/language/LanguageEditor.class.php

```diff
@@ -141,25 +139,32 @@ public function export($packageIDArray = array(), $exportCustomValues = false) {
 		
 		// sort categories
 		ksort($items);
-		
+
+		// write language file
+		$filename = FileUtil::getTemporaryFilename($this->languageCode, '.xml');
+
+		$xml = new XMLWriter();
+		$xml->beginDocument('language', 'http://www.woltlab.com', 'http://www.woltlab.com/XSD/maelstrom/language.xsd', array('languagecode' => $this->languageCode, 'languagename' => $this->languageName, 'countrycode' => $this->countryCode));
+
 		foreach ($items as $category => $categoryItems) {
 			// sort items
 			ksort($categoryItems);
-			
-			// category header
-			echo "\t<category name=\"".$category."\">\n";
+
+			$xml->startElement('category', array('name' => $category));
 			
 			// items
 			foreach ($categoryItems as $item => $value) {
-				echo "\t\t<item name=\"".$item."\"><![CDATA[".StringUtil::escapeCDATA($value)."]]></item>\n";
+				$xml->writeElement('item', StringUtil::escapeCDATA($value), array('name' => $item));
```

Comment:
`writeCData` is already used inside [`writeElement()`](https://github.com/WoltLab/WCF/blob/master/wcfsetup/install/files/lib/util/XMLWriter.class.php#L133) but you're right `escapeCDATA` is unnecessary.